packages/commonwealth/client/scripts/state/index.ts

```diff
@@ -58,9 +55,6 @@ export interface IApp {
   threads: ThreadsController;
   threadUniqueAddressesCount: ThreadUniqueAddressesCount;
   comments: CommentsController;
-  reactions: ReactionsController;
-  threadReactions: ThreadReactionsController;
-  reactionCounts: ReactionCountsController;
```

Comment:
Some of this `reactionCounts` controller state is merged with comments. We can remove this when the comments controller is migrated to react query (same reason as above)